packages/core/src/components/application/application.scss

```
@@ -19,6 +20,38 @@
 
   @include component.ix-component;
 
+  .skip-link {
+    @include fonts.typography-body;
+
+    position: absolute;
+    inset-block-start: -100%;
+    inset-inline-start: calc(
+      var(--ix-safe-area-inset-left, 0rem) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)